src/clj/ataru/forms/form_access_control.clj

```diff
@@ -99,13 +99,20 @@
         (form-store/create-form-or-increment-version!
          (assoc form :deleted true))))))
 
-(defn get-forms-for-editor [session organization-service]
+(defn- get-forms-as-ordinary-user [session virkailija-tarjonta-service organization-oids]
+  (let [forms-with-organization-oids                              (form-store/get-forms organization-oids)
+        forms-using-tarjonta-keys                                 (set (keys (.get-forms-in-use virkailija-tarjonta-service (-> session :identity :username))))
```

Comment:
Tässä voisi `require`:ta `get-forms-in-use` funktion ja käyttää sitä Java interopin sijaan.